src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs

```diff
@@ -115,12 +114,33 @@ private static string GetString(char[] word)
             return new String(word, 0, length);
         }
 
+        private event EventHandler<CameraDeviceConnectionChangedEventArgs> _deviceConnectionChanged;
         /// <summary>
         /// An event that occurs when camera device is connected or disconnected.
         /// </summary>
         /// <since_tizen> 9 </since_tizen>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public event EventHandler<CameraDeviceConnectionChangedEventArgs> DeviceConnectionChanged;
+        public event EventHandler<CameraDeviceConnectionChangedEventArgs> DeviceConnectionChanged
+        {
+            add
+            {
+                if (_deviceConnectionChanged == null)
+                {
+                    RegisterDeviceConnectionChangedCallback();
+                }
+
+                _deviceConnectionChanged += value;
+            }
+            remove
+            {
+                _deviceConnectionChanged -= value;
+
+                if (_deviceConnectionChanged == null)
+                {
+                    UnregisterDeviceConnectionChangedCallback();
+                }
```

Comment:
How about unregistering callback first?

                if (_deviceConnectionChanged == value)
                {
                    UnregisterDeviceConnectionChangedCallback();
                }

                _deviceConnectionChanged -= value;

Reply:
This code intend to remove event first, and then if there's no added event(that is, `_deviceConnectionChanged` is null), unregister native callback also.
If we call `UnregisterDeviceConnectionChangedCallback()` when `_deviceConnectionChanged == value` , the remained event cannot be invoked anymore.

Reply:
Okay.